convoy.json.example

```diff
@@ -0,0 +1,80 @@
+{
+  "database": {
+    "environment": "dev",
+    "dsn": "mongodb://root:rootpassword@localhost:27017/convoy?authSource=admin"
+  },
+  "sentry": {
+    "dsn": "<insert-sentry-dsn>"
+  },
+  "queue": {
+    "type": "redis",
+    "redis": {
+      "dsn": "redis://localhost:8379"
```

Comment:
This connection will also fail in the docker network because `localhost` will not resolve to the actual redis server container
```suggestion
      "dsn": "redis://redis_server:8379"
```